testing/endtoend/components/validator.go

```diff
@@ -177,6 +177,7 @@ func (node *ValidatorNode) saveConfig() (string, error) {
 
 // Start starts a validator client.
 func (v *ValidatorNode) Start(ctx context.Context) error {
+	fmt.Println("trying to start a validator node")
```

Comment:
Is this intentional?

Reply:
totally missed removing it, was having trouble finding why certain failures were happening in e2e so i was marking this.